strapi/strapi
# [DEITS] Transfer enhancements

### What does it do?

- Update the transfer progress logic to make it more customizable
- Update the transfer table to sort aggregate items by alphabetical order
- Update the transfer table to display more aggregate information (assets by extension, entities by model type)
- Add a new type for the assets
- Fix the engine tests by providing accurate & valid data for each stage stream


### How to test it?

- Run the unit test, everything should work properly
- Run an export, everything should be exported like before + you should see the new changes in the CLI transfer table
